autoload/arduino.vim

```diff
@@ -54,6 +54,9 @@ function! arduino#InitializeConfig()
   if !exists('g:arduino_run_headless')
     let g:arduino_run_headless = executable('Xvfb') ? 1 : 0
   endif
+  if !exists('g:arduino_user_installation')
+    let g:arduino_user_installation = 0
```

Comment:
If you want, you could default this to 1 if `s:FileExists($HOME . ".arduino15")`. If you do, you should probably pull `$HOME . ".arduino15"` out into a constant or a function

Reply:
Done :)